wake-word-voice-assistant/m5stack-atom-echo.yaml

```diff
@@ -1 +1,352 @@
-<<: !include ../voice-assistant/m5stack-atom-echo.yaml
+substitutions:
+  name: m5stack-atom-echo
+  friendly_name: M5Stack Atom Echo
+  micro_wake_word_model: okay_nabu # hey_jarvis and hey_mycroft are also supported
```

Comment:
Is alexa not supported?
Are we able to pull this config (which users will likely want to change) up to the config section visible to users in ESPHome?

Reply:
Alexa works fine, and can be changed by adding a line in the ESPHome level config 👍

Reply:
I haven't investigated whether the wake word model can be changed dynamically at run time. None of the other firmwares on here do that and I've been aiming for feature parity in this PR. 
Might be something for a future PR. Along with optional replies on another media player